opta/flyte.yaml

```diff
@@ -0,0 +1,161 @@
+environments:
+  - name: default
+    path: "./env.yaml" # NOTE: relative path to environment
+    variables:
+      region: {region}
+      account_id: {account_id}
+name: service-flyte
+modules:
+  - name: postgres
+    type: aws-postgres
+  - name: s3
+    type: aws-s3
+    bucket_name: "{parent_name}-{layer_name}"
+  - name: notifcationsQueue
+    type: aws-sqs
+  - name: schedulesQueue
+    type: aws-sqs
+  - name: topic
+    type: aws-sns
+    sqs_subscribers:
+      - "${{module.notifcationsQueue.queue_arn}}"
+  - name: adminflyterole
+    type: aws-iam-role
+    extra_iam_policies:
+      - "arn:aws:iam::aws:policy/CloudWatchEventsFullAccess"
+#      - "arn:aws:iam::{vars.account_id}:policy/{env}-{env}-awsses-sender" # Uncomment out for SES
+    allowed_k8s_services:
+      - namespace: "*"
+        service_name: "*"
+    links:
+      - s3: [ "write" ]
+      - notifcationsQueue
+      - schedulesQueue
+      - topic
+  - name: userflyterole
+    type: aws-iam-role
+    extra_iam_policies:
+      - "arn:aws:iam::aws:policy/CloudWatchEventsFullAccess"
+#      - "arn:aws:iam::{vars.account_id}:policy/{env}-{env}-awsses-sender" # Uncomment out for SESre, change the templating
+    allowed_k8s_services:
+      - namespace: "*"
+        service_name: "*"
+    links:
+      - s3: [ "write" ]
+  - type: helm-chart
+    chart: "../helm" # NOTE: relative path to chart
+    namespace: flyte
+    timeout: 600
+    create_namespace: true
+    values_file: "../helm/values-eks.yaml" # NOTE: relative path to values yaml
+    values:
+      postgres:
+        enabled: false
+      db:
+        database:
+          port: 5432
+          username: "${{module.postgres.db_user}}"
+          host: "${{module.postgres.db_host}}"
+          dbname: "${{module.postgres.db_name}}"
+      common:
+        ingress:
+          albSSLRedirect: false
+          host: "{parent.domain}"
```

Comment:
Any chance this can be cleaned up to keep referencing opta-parameters the same way across the file?
`{something}` vs `${{something}}` ? perhaps a tracking bug on opta project side

Reply:
Just filed a jira for it!